command_list.md

```diff
@@ -0,0 +1,116 @@
+# Account
+```
+rucio account list
+rucio account add
+rucio account show
+rucio account remove
+rucio account history
+rucio account update
+rucio account ban add
+rucio account ban remove
+rucio account attribute list
+rucio account attribute add
+rucio account attribute remove
+rucio account limit list
+rucio account limit add
+rucio account limit remove
+rucio account identity list
+rucio account identity add
+rucio account identity remove
```

Comment:
I think bundling identities with accounts is interesting. I’ll think it over, but my initial impression is positive.